src/Movie-details/MovieDetails.js

```diff
@@ -3,41 +3,65 @@ import Billboard from '../Billboard/Billboard'
 import Ratings from '../Ratings/Ratings'
 import Details from '../Details/Details'
 import Videos from '../Videos/Videos'
-import { getMovieDetails } from '../apiCalls'
+import Comments from '../Comments/Comments'
+import CommentForm from '../CommentForm/CommentForm'
+import { getMovieDetails, getComments, postComment } from '../apiCalls'
 import '../scss/_MovieDetails.scss'
 
 class MovieDetails extends Component {
   constructor(props) {
     super(props)
     this.state = {
-      movieId: this.props.movieId,
```

Comment:
I did use movieId from state in my solo iteration, but this is an easy fix! I will just need to change it up to use props instead of state. I like that you updated this though after learned we are not to use props in state!